controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -16,92 +16,110 @@
 from controller.tests import utils
 
 
-class AbstractControllerTest(unittest.TestCase):
+class BaseControllerSetUp(unittest.TestCase):
 
     def setUp(self):
         patch("controller.array_action.array_mediator_xiv.XIVArrayMediator._connect").start()
         detect_array_type_patcher = patch("controller.controller_server.csi_controller_server.detect_array_type")
         self.detect_array_type = detect_array_type_patcher.start()
         self.detect_array_type.return_value = "a9k"
         self.addCleanup(detect_array_type_patcher.stop)
+        self.fqdn = "fqdn"
+        self.mediator = XIVArrayMediator("user", "password", self.fqdn)
+        self.mediator.client = Mock()
+
+        self.storage_agent = MagicMock()
+        self.storage_agent.get_mediator.return_value.__enter__.return_value = self.mediator
+
+        self.servicer = ControllerServicer(self.fqdn)
+
+        self.request = Mock()
+
+
+class CommonControllerTest:
 
     @abc.abstractmethod
-    def get_create_object_method(self):
+    def get_tested_method(self):

```

Comment:
I know it's not part of this PR, but if you're already changing here, could you try to add a `@property` decorator for these two abstract methods (and remove the `_get` prefix)? it will avoid the `self.get_tested_method()(...)` train wrecks

Reply:
I tried for half an hour, didnt succeed opened ticket - CSI-2372

Reply:
ok, thanks